src/store/modules/activity_daily.ts

```diff
@@ -25,6 +25,20 @@ function timeperiodToStr(tp: TimePeriod): string {
   return [start, end].join('/');
 }
 
+function get_buckets_by_type(buckets, type) {
```

Comment:
Nice, but duplicated code (also in `modules/buckets.js`). 

Move all of it there?

Reply:
That's the plan, hence the comment below

// TODO: re-use modules/buckets.js instead of doing a second getBuckets call

Reply:
Ah, could still have done an import for this particular helper function in the meantime, but whatever if you're going to fix it before merging.